scaleway_functions_python/local/serving.py

```diff
@@ -128,9 +128,9 @@ def resp_record_to_flask_response(
         self, record: "hints.ResponseRecord"
     ) -> "FlaskResponse":
         """Transform the ReponseRecord into an http reponse."""
-        body = record.get("body", "")
+        body: Union[str, bytes] = record.get("body", "")
         if record.get("isBase64Encoded") and body:
-            body = b64decode(body.encode("utf-8"), validate=True).decode("utf-8")
+            body = b64decode(cast(str, body).encode("utf-8"), validate=True)
```

Comment:
For reviewers, this isn't the most legible code. The type annotations were added to satisfy mypy, the cast it there for the type-checker but does not do anything at runtime.